cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -18,6 +18,10 @@
 
 #ifdef MODULE_ESP_WIFI
 
+#if defined(MODULE_ESP_NOW) && defined(MODULE_ESP_WIFI_AP)
+#error ESP_NOW and ESP_WIFI_AP can not be used at the same time
+#endif
+
```

Comment:
```suggestion
```

The conflict can be checked already by the make system. Just add the following lines to `cpu/esp_common/Makefile.features`
```
FEATURES_CONFLICT += esp_wifi_ap:esp_now
FEATURES_CONFLICT_MSG += "ESP_NOW and ESP_WIFI_AP can not be used at the same time."
```

Reply:
Solved, added a `FEATURE_REQUIRED += esp_now` when `esp_now` is used.